.github/workflows/ci.yml

```diff
@@ -26,8 +26,10 @@ jobs:
         with:
           version: "14.0"
       - name: Setup rust toolchain
-        run: rustup show
-
+        uses: ./.github/actions/rust-toolchain
+        with:
+          toolchain: 1.78.0
```

Comment:
Should this be 1.79?

Reply:
I think we can, I was trying to match the ADO req